query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -27,6 +27,9 @@ func init() {
 		// For this rule to take effect the appropriate capabilities must be
 		// added AND feature flags must be enabled.
 		// PushDownWindowAggregateRule{},
+
+		// Currently no guards on this, will induce a panic if enabled.
+		// PushDownGroupAggregateRule{},
```

Comment:
If none of the feature flags are turned on for this rule, isn't it safe to register it? I must be missing something.

Reply:
It is, just being extra cautious. I thought I would make a separate PR when when It's confirmed we are ready to start pushing this out to production, monitoring it etc.